bd_file = 'files/photocoll.sqlite'
icon = 'files/icon.ico'

# UI файлы
main_window = 'files/main_window.ui'


# UI файлы АЛЬБОМЫ
new_album = 'files/album/new_album.ui'

del_album = 'files/album/del_album.ui'
edit_album = 'files/album/edit_album.ui'
del_image = 'files/album/del_image.ui'
add_image = 'files/album/add_image.ui'

all_img_album = 'files/album/all_img_album.ui'


# UI файлы ТЕГИ
new_tag = 'files/tags/new_tag.ui'
del_tag = 'files/tags/del_tag.ui'
edit_tag = 'files/tags/edit_tag.ui'

# Статусы
new_album_text = 'Новый альбом создан'
del_album_text = 'Альбом удален'


new_image_text = 'Новое изображение добавлено'
del_image_text = 'Изображение удалено'


new_tag_text = 'Новый тег создан'
del_tag_text = 'Тег удален'
edit_tag_text = 'Тег изменен'


